### PR TITLE
feat: Add new reactions annotated for genes related to Glycolysis pathway 

### PR DESCRIPTION
**Pull request by feiranl**
#### Main improvements in this PR:
* Add 87 new mets, 68 new reactions for genes related to Glycolysis pathway according to the #507 

**I hereby confirm that I have:**

- [x] Tested my code on my own computer for running the model
- [x] Selected `develop` as a target branch
- [x] Any removed reactions and metabolites have been moved to the corresponding deprecated identifier lists

### CODE REVIEW

File: code/modelCuration/GlycolysisCuration.m [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+% metID metNames	metFormulas	metCharges	compartments	metKEGGID	metPubChemID	metChEBIID	metMetaCycID	metMetaNetXID
+metNocomp = cellfun(@(s) s(1:8), MetStruct.MetID(idx,1), 'UniformOutput', false);
+structure.mets = [structure.mets;MetStruct.MetID(idx,1)];
+structure.metBiGGID = [structure.metBiGGID;MetStruct.BiGGID(idx,1)];
+structure.metKEGGID = [structure.metKEGGID;MetStruct.KEGG(idx,1)];
+structure.metsNoComp = [structure.metsNoComp;metNocomp];
+structure.metHMDBID = [structure.metHMDBID;MetStruct.BiGGID(idx,1)];
+structure.metChEBIID = [structure.metChEBIID;MetStruct.CHEBI(idx,1)];
+structure.metPubChemID = [structure.metPubChemID;MetStruct.BiGGID(idx,1)];
+structure.metLipidMapsID = [structure.metLipidMapsID;MetStruct.CHEBI(idx,1)];
```
Comment (haowang-bioinfo) [inline code/modelCuration/GlycolysisCuration.m:49]: this line probably should be modified, now ChEBI ids were also updated to the column of Lipidmap ids

**Comment by haowang-bioinfo** [comment]
very good progress!

File: code/modelCuration/GlycolysisCuration.m (hunk 1 above)
Comment (feiranl) [reply code/modelCuration/GlycolysisCuration.m:49]: They are all empty. but it is better to rename it to not cause misunderstanding.

Comment (feiranl) [reply code/modelCuration/GlycolysisCuration.m]: Thanks for identifying this. Changed in the latest commit.

**Review by haowang-bioinfo — APPROVED** [review]
looks good

**Pull request merged**